tests/pytest.ini

```diff
@@ -1,5 +1,8 @@
 [pytest]
 markers =
+    unit_test: marks tests as unit-level tests (select with '-m "unit_test"')
+    integration_test: marks tests as integration-level tests (select with '-m "integration_test"')
+    system_test: marks tests as system-level tests (select with '-m "system_test"')
```

Comment:
Would `unit`, `integration` and `system` also be an option? We anyways type out the path to the marker as `pytest.mark.unit`, so maybe the `_test` part is a bit redundant? Just an idea though.

Reply:
Yep, I am open to that too. @trbromley  any opinion here? I'm fine either way.

Reply:
And done. Thanks folks.